source/sourcing_and_processing_data/processors/normalize_date.rst

```diff
@@ -0,0 +1,97 @@
+Normalize date processor
+========================
+
+This processor allows to parse a date which format is not automatically understood by the platform.
+
+Example: if a date is in the dd/mm/YYYY French format (for instance 09/10/2017 for October 9th 2017), the platform will by default parse this date using the mm/dd/YYYY US format (09/10/2017 will then be understood by the platform as September 10th 2017).
```

Comment:
I think that we want to avoid talking about french or US format, we support ISO 8601 format which is YYYY-mm-dd others formats are ambiguous.